cel/cel_test.go

```diff
@@ -1263,3 +1264,21 @@ func TestResidualAst_Modified(t *testing.T) {
 		}
 	}
 }
+
+func TestDeclareContextProto(t *testing.T) {
+	descriptor := new(tpvpb.TestProtoValue).ProtoReflect().Descriptor()
+	option := DeclareContextProto(descriptor)
+	env, err := NewEnv(option)
+	if err != nil {
+		t.Fatalf("NewEnv(DeclareContextProto(%v)) failed: %s", descriptor, err)
+	}
+	expression := "single_int64 == 1 && single_double == 1.0 && single_bool == true && single_string == '' && single_nested_message == cel.testdata.TestProtoValue.NestedMessage{} && single_nested_enum == cel.testdata.TestProtoValue.NestedEnum.FOO && single_duration == duration('5s') && single_timestamp == timestamp('1972-01-01T10:00:20.021-05:00') && single_any == google.protobuf.Any{} && repeated_int32 == [1,2] && map_int64_string == {0: ''}"
```

Comment:
Could you please break up the expression onto multiple lines? I would recommend using the backtick ` for multiline string quoting

Reply:
Done.